src/shim.js

```diff
@@ -1,35 +1,6 @@
 import webpage from "webpage";
 import system from "system";
-
-/**
- * Ensure that Function has bind() method (PhantomJS version <= 1.9 support)
- * This is a Polyfill replacement from MDN.
- * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill}
- */
-/* eslint-disable no-extend-native, consistent-this, require-jsdoc, no-empty-function, no-invalid-this */
-if (!Function.prototype.bind) {
-    Function.prototype.bind = function (oThis) {
-        if (typeof this !== 'function') {
-            throw new TypeError('Function.prototype.bind - what is trying to be bound is not callable');
-        }
-
-        let self = this;
-        let aArgs = Array.prototype.slice.call(arguments, 1);
-        function NoopFunction() {}
-        function boundFunction() {
-            return self.apply((this instanceof NoopFunction ? this : oThis), aArgs.concat(Array.prototype.slice.call(arguments)));
-        }
-
-        if (this.prototype) {
-            NoopFunction.prototype = this.prototype;
-        }
-        boundFunction.prototype = new NoopFunction();
-
-        return boundFunction;
-    };
-}
-
-/* eslint-enable no-extend-native, consistent-this, require-jsdoc, no-empty-function, no-invalid-this */
+import {ensureFunctionBind} from "./shim_support";
```

Comment:
So I don't think you need to export a function. Just put the logic in the other file as you had before and doing `import from "./other_file"` should work. 

I can also do it after my meeting.